lib/api/controllers/readController.js

```diff
@@ -25,32 +25,11 @@ module.exports = function ReadController (kuzzle) {
     return kuzzle.services.list.readEngine.count(requestObject);
   };
 
-  this.listCollections = function (requestObject, context) {
+  this.listCollections = function (requestObject) {
```

Comment:
hmm well, I don't really agree with that... If a user can see a collection that he is not allowed to use, it can be a security issue. It's safer to show only the things he can use.

Anyway, if we decide to change that:
1. this has nothing to do with actual PR (it's not closure-related) and should be proposed in another PR
2. to be consistent, we have to to the same with listIndexes action.